brewcenter_api/brew_data/models/yeast.py

```diff
@@ -1,16 +1,32 @@
+from django.contrib.auth.models import User
 from django.db import models
 
+from brewcenter_api.accounts.models import Token
+
+
 class YeastType(models.Model):
     """
     Defines a Yeast type. Examples include:
     - Ale
     - Lager
     """
     name = models.CharField(max_length=255)
+
     def __str__(self):
         return self.name
 
 
+class YeastTypeRecommendation(models.Model):
+    model = models.ForeignKey(YeastType, null=True, blank=True)
```

Comment:
The model field should probably be required (not nullable or blank), since it contains all the data being suggested. Unless you were thinking this would be how one suggests to delete a value?